raiden/raiden_service.py

```diff
@@ -447,6 +447,7 @@ def start(self):
             database_path=self.database_path,
             serializer=serialize.JSONSerializer(),
         )
+        storage.update_version()
```

Comment:
This should be done by the upgrade manager, no?

Reply:
What if there are no prior databases to migrate? i did not want to introduce a regression so we do it explicity here.